releases/release-6.6.0.md

```diff
@@ -0,0 +1,416 @@
+
+    更多信息，请参考[用户文档](/sql-plan-management.md#根据历史执行计划创建绑定)。
+
+* Support configuring `SURVIVAL_PREFERENCE` for [placement rules in SQL](/placement-rules-in-sql.md) [#38605](https://github.com/pingcap/tidb/issues/38605) @nolouch[https://github.com/nolouch] **tw@qiancai**
+
+    `SURVIVAL_PREFERENCES` provides data survival preference settings to increase the disaster survivability of data. By specifying `SURVIVAL_PREFERENCE`, you can control the following:
+
+    - For TiDB clusters deployed across regions, when a region with the specified databases or tables fails, another region can provide the service.
+    - For TiDB clusters deployed in a single region, when an availability zone with the specified databases or tables fails, another availability zone can provide the service.
+
+     For more information, see [documentation](/placement-rules-in-sql.md#survival-preference).
+
+### Security
+
+* TiFlash supports automatic rotations of TLS certificates [#5503](https://github.com/pingcap/tiflash/issues/5503) @[ywqzzy](https://github.com/ywqzzy) **tw@qiancai**
+
+    For a TiDB cluster with encrypted data transmission between components enabled, when a TLS certificate of TiFlash expires and needs to be reissued with a new one, the new TiFlash TLS certificate can be automatically loaded without restarting the TiDB cluster. The rotation of a TLS certificate between components within a TiDB cluster does not affect the use of the TiDB cluster, which ensures the cluster high availability.
+
+    For more information, see [documentation](/enable-tls-between-components.md).
+
+### 可观测性
+
+* 快速绑定执行计划 [#781](https://github.com/pingcap/tidb-dashboard/issues/781) @[YiniXu9506](https://github.com/YiniXu9506) **tw@ran-huang**
+
+    TiDB 的执行计划快速绑定功能：允许用户在 TiDB Dashboard 中一分钟内完成 SQL 与特定计划的绑定。
+
+    通过提供友好的界面简化在 TiDB 上绑定计划的过程，减少计划绑定过程的复杂性提高用户体验，提高计划绑定过程的效率。
+
+    更多信息，请参考[用户文档](/dashboard/dashboard-statement-details.md)。
+
+* 为执行计划缓存增加告警 [#issue号](链接) @[qw4990](https://github.com/qw4990) **tw@TomShawn**
+
+    当执行计划无法进入执行计划缓存时， TiDB 会通过 warning 的方式说明其无法被缓存的原因， 降低诊断的难度。例如：
+
+    ```sql
+    mysql> prepare st from 'select * from t where a<?';
+    Query OK, 0 rows affected (0.00 sec)
+
+    mysql> set @a='1';
+    Query OK, 0 rows affected (0.00 sec)
+
+    mysql> execute st using @a;
+    Empty set, 1 warning (0.01 sec)
+
+    mysql> show warnings;
+    +---------+------+----------------------------------------------+
+    | Level   | Code | Message                                      |
+    +---------+------+----------------------------------------------+
+    | Warning | 1105 | skip plan-cache: '1' may be converted to INT |
+    +---------+------+----------------------------------------------+
+    ```
+
+    上述例子中， 优化器进行了非 INT 类型到 INT 类型的转换，产生的计划可能随着参数变化有风险，因此不缓存。
+
+    更多信息，请参考[用户文档](/sql-prepared-plan-cache.md#prepared-plan-cache-诊断)。
```

Comment:
```suggestion
* 为执行计划缓存增加告警 @[qw4990](https://github.com/qw4990) **tw@TomShawn**
* Add warning for caching execution plans @[qw4990](https://github.com/qw4990) **tw@TomShawn**

    当执行计划无法被缓存时，TiDB 会通过告警的方式提示该计划无法被缓存的原因，降低诊断的难度。例如：
    When an execution plan cannot be cached, TiDB indicates the reason in warning to make diagnostics easier. For example:

    ```sql
    mysql> PREPARE st FROM 'SELECT * FROM t WHERE a<?';
    Query OK, 0 rows affected (0.00 sec)

    mysql> SET @a='1';
    Query OK, 0 rows affected (0.00 sec)

    mysql> EXECUTE st USING @a;
    Empty set, 1 warning (0.01 sec)

    mysql> SHOW WARNINGS;
    +---------+------+----------------------------------------------+
    | Level   | Code | Message                                      |
    +---------+------+----------------------------------------------+
    | Warning | 1105 | skip plan-cache: '1' may be converted to INT |
    +---------+------+----------------------------------------------+
    ```

    在以上例子中，优化器进行了非 INT 类型到 INT 类型的转换，产生的执行计划可能随着参数变化而存在风险，因此 TiDB 不缓存该计划。
    In the preceding example, the optimizer converts a non-INT type to an INT type, and the execution plan might change with the change of the parameter, so TiDB does not cache the plan.

    更多信息，请参考[用户文档](/sql-prepared-plan-cache.md#prepared-plan-cache-诊断)。
    For more information, see [documentation](/sql-prepared-plan-cache.md#diagnostics-of-prepared-plan-cache).
```